php/example_code/sns/EnableRule.php~master

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+require 'vendor/autoload.php';
+
+use Aws\CloudWatchEvents\CloudWatchEventsClient;
+use Aws\Exception\AwsException;
+
+/**
+ * Enable Rule
```

Comment:
Nit: same as on DeleteRule.php, specifying that this is for CloudWatch Events rules instead of IoT rules would help.
```suggestion
 * Enable Rule for CloudWatch Events
```